src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -23,13 +23,13 @@
 import com.google.api.services.youtube.YouTube;
```

Comment:
This should come before importing youtube.model.CommentThreadListResponse (prior lexicographical order)